src/things/comment.rs

```diff
@@ -204,6 +204,12 @@ impl Comment {
             debug!("Skipping due to `before` filter ({})", config.before);
             return true;
         }
+        if let Some(only_subreddits) = &config.only_subreddits {
+            if !only_subreddits.is_empty() && !only_subreddits.contains(&self.subreddit) {
```

Comment:
nit: this should be the same logic if you remove `!only_subreddits.is_empty()`. same with the rest of the diff.